### ISSUE TITLE
Feedback Form

### ISSUE THREAD

**Issue by CliffordD** [rory/osm-irish-townlands#94]
Would it be possible to add a feedback tool on townlands.ie, for example to report an issue with the map?  When a visitor is looking at the townland they may notice a mistake or have information to add, it would be useful to capture this feedback and in turn improve the map.

**Comment by rory** [comment]
Good idea. But, where should the feedback go? Maybe an OSM note? or a post to the mailing list?

**Comment by CliffordD** [comment]
I'm not sure, not all feedback will be map related I'm sure. Although quite a few map notes aren't helpful either.  There are a few mappers who might not be on the mailing list, so maybe Map Note would be the most visible?
